src/core/backbone_router/bbr_manager.hpp

```diff
@@ -133,6 +133,34 @@ class Manager : public InstanceLocator, private NonCopyable
      */
     BackboneTmfAgent &GetBackboneTmfAgent(void) { return mBackboneTmfAgent; }
 
+    /**
+     * This method sends BB.qry on the Backbone link.
+     *
+     * @param[in]  aDua     The Domain Unicast Address to query.
+     * @param[in]  aRloc16  The RLOC16 in the RLOC16 TLV of BB.qry, or Mac::kShortAddrInvalid to not have RLOC16 TLV.
```

Comment:
RLOC16 TLV is optional for BB.qry and would be skipped for DAD bq.
```suggestion
     * @param[in]  aRloc16  The short address of the address resolution initiator or `Mac::kShortAddrInvalid` for DUA DAD.
```